src/AdminConsole/Components/Pages/App/Settings/AuthenticationConfiguration/AuthenticationConfigurationForm.razor

```diff
@@ -8,43 +8,49 @@
     <ValidationSummary/>
     <div class="sm:w-full sm:max-w-md">
         <label class="block text-sm font-medium leading-6 text-gray-900 my-2" for="Purpose">Purpose:</label>
-        <InputText @bind-Value="@FormModel!.Purpose" id="Purpose" readonly="@IsPurposeReadOnly" class="text-input"/>
+        <InputText id="Purpose" @bind-Value="@FormModel!.Purpose" readonly="@IsPurposeReadOnly" class="text-input"/>
+
         <label class="block text-sm font-medium leading-6 text-gray-900 my-2" for="UserVerification">User Verification:</label>
         <InputSelect id="UserVerification" @bind-Value="@FormModel.UserVerificationRequirement" TValue="UserVerificationRequirement" class="form-select">
             @foreach (var uv in Enum.GetValues<UserVerificationRequirement>())
             {
                 <option value="@uv">@uv</option>
             }
         </InputSelect>
+
         <label class="block text-sm font-medium leading-6 text-gray-900 my-2" for="Seconds">Seconds:</label>
         <InputNumber id="Seconds" @bind-Value="@FormModel.Seconds" min="0" class="text-input"/>
+
+        <label class="block text-sm font-medium leading-6 text-gray-900 my-2" for="Hints">Credential hints:</label>
+        <InputText id="Hints" @bind-Value="@FormModel.HintString" class="text-input" />

```

Comment:
You can use `DisplayName` to make it display `Hints` instead of `HintString`

Reply:
It doesn't actually display `HintString` anywhere. Where do you see it?

Reply:
Ah I see. You mean in the validation message.